README.md

```diff
@@ -47,3 +40,95 @@ sudo ./check-payload scan operator \
 IMAGE=some.registry.location/check-payload
 podman  run --privileged -ti -v /:/myroot $IMAGE scan node --root /myroot
 ```
+
+## How it works
+
+`check-payload` gathers container images from OpenShift release payloads or
+container images. The scanner can be used on a RHEL or RHCOS node to scan the
```

Comment:
`s/can be used/can also be used/`?

Reply:
I mean, adding `also` makes it more clear that the tool can be used to either scan remote images, or local node files.